vocabularies/Org.OData.Core.V1.xml

```diff
@@ -457,6 +457,10 @@
         <Annotation Term="Core.Description" String="The qualified name of an action in scope." />
       </TypeDefinition>
 
+      <TypeDefinition Name="QualifiedOperationName" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="The qualified name of an action or function in scope. For a Function, the qualified function name may optionally be followed by parentheses containing the comma-separated list of parameter types, in the order of their definition, to identify a single function overload." />

```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The qualified name of an action or function in scope, optionally followed by parentheses containing the comma-separated list of parameter types, in the order of their definition, to identify a single function overload." />
```
Otherwise it sounds to me as if the parenthesised expression _followed_ the `QualifiedOperationName` rather than being part of it.

Reply:
Bound actions only support overloading by binding type, so for this usage there's no value in appending the parameters for the action case; only the function case.

So how about: "The qualified name of a bound action or function in scope. The qualified function name may include trailing parentheses containing the comma-separated list of parameter types, in the order of their definition, to identify a single function overload."

Reply:
I did not mean to allow overloading of actions, I thought this would be clear from:
> optionally followed by parentheses containing the comma-separated list of parameter types, in the order of their definition, to identify a single _function_ overload.

But your suggestion is clearer than mine.